quickstarts/dotnet/csharp/config.yml

```diff
@@ -5,22 +5,33 @@ description: |
   ## Comprehensive monitoring quickstart for C#
   With our C# integration, you can find and address performance bottlenecks while operating in a live development environment. Monitor your app via our dashboard quickstarts and ensure you're delivering the best user experience.
   
-  ## Why monitor C#?
-  - Focus on the issues that affect your critical business transactions.
-  - Resolve problems quickly.
-  - Gain knowledge on your system's behavior in advance.
-  - Learn the whole story of performance errors, from beginning to end.
-  - Use traces to connect latency issues to errors, getting you to the root cause as quickly as possible.
-  - See the filename and line number from the C# stack trace so you never have to guess. To reduce noise, logically filter and aggregate C# exceptions.
-  - Scalable error monitoring without affecting production flow.
-  - Maintain low latency with fast throughput.
-  - Get meaningful data about C# errors from your dashboard.
+  ## Common C# performance errors
+  Here are the common C# performance errors that you can handle with New Relic integration:
   
-  ## End-to-end visibility into your C# operations
+  Memory management: C# uses automatic memory management, which takes the burden of manual allocation from developers. However, C# can sometimes run into memory error thereby inhibiting its memory management performance. Our integration offers the best solution to fix that.
```

Comment:
Perhaps these should be in a list rather than a paragraph. It's looking a bit odd in the preview. https://newrelic.com/instant-observability/preview?pr=1575&quickstart=dotnet%2Fcsharp#:~:text=best%20user%20experience.-,Common%20C%23%20performance%20errors,-Here%20are%20the